zap/src/test/java/org/zaproxy/zap/extension/anticsrf/ExtensionAntiCSRFUnitTest.java

```diff
@@ -293,7 +293,7 @@ void shouldGetTokensFromInputFieldsIfIdAndNameIsAKnownTokenWithDifferentCase() {
         }
     }
 
-    /** Tests for {@link ExtensionAntiCSRF#generateForm(HttpMessage)}. */
+    /** Tests for {@link ExtensionAntiCSRF#generateForm(HttpMessage, String)}. */
```

Comment:
These changes should not be needed (the existing method calls the new), it would also be good to test the new behaviour.